winui/DropDown-Color-Palette/Getting-Started.md

```diff
@@ -146,7 +146,7 @@ SfDropDownColorPalette sfDropDownColorPalette =  new SfDropDownColorPalette();
 {% endhighlight %}
 {% endtabs %}
 
-![Dropdown Color Palette control with theme and standard color items](Getting-Started_images/ThemeColorPanel.png)
+![WinUI Dropdown Color Palette control with theme and standard color items](getting-started_images/winui-color-palette-theme-color-panel.png)
```

Comment:
Change image name under in Folder.
<img width="778" alt="image" src="https://github.com/syncfusion-content/winui-docs/assets/44014283/180b700c-9f93-4249-9535-685c762703a3">

Reply:
Changed